### DISCUSSION TITLE
Do I need to update?

### DISCUSSION THREAD

**Discussion by itrn57** [j-kaltes/Juggluco#25] [Q&A]
Hi, thank you for your great work.
Recently I noticed this banner on the download page: 
"WARNING: If you use the WearOS version of Juggluco, you should update both the WearOS and the Phone version to 4.15.0 or above"
Does it mean the update is mandatory and if so, why it is?
I use v. 4.10.1 on both watch and phone and don't see any issues apart from occasional/usual distonnect/reconnect.
Do I need to update?

**Comment by j-kaltes** [comment]
I think that you can judge yourself whether you need to update. I only wanted to convey that something in the connection between watch and phone has changed, so you can better update both when you do update. But why else are you looking at the download page?
When you don't update, you will miss all changes since 4.10.1 as described at: https://www.juggluco.nl/Juggluco/changes.html

**Reply by itrn57** [reply]
Thank you for a quick reply and explanation. I understant your point. I always look at the download page to check what is new with the app.
